common/attest_plugin.c

```diff
@@ -49,8 +49,8 @@ const char* OE_OPTIONAL_CLAIMS[OE_OPTIONAL_CLAIMS_COUNT] = {
 static oe_plugin_list_node_t* verifiers = NULL;
 
 // UUID for all OE reports generated by oe_get_report().
-static const oe_uuid_t _sgx_ecdsa_report_uuid = {
-    OE_FORMAT_UUID_SGX_ECDSA_P256_REPORT};
+static const oe_uuid_t _oe_report_remote_uuid = {
```

Comment:
Can we make the variable name match the define name?
```suggestion
static const oe_uuid_t _oe_format_uuid_legacy_report_remote = {
```

Reply:
Thanks!